src/containers/Sidebar/Index.tsx

```diff
@@ -0,0 +1,78 @@
+import { AiFillHome } from 'solid-icons/ai'
+import { BiSolidExtension } from 'solid-icons/bi'
+import { IoSettingsSharp } from 'solid-icons/io'
+import { Component } from 'solid-js'
+import { NAVIGATION } from '@interfaces/enums'
+
+export interface IProps {
+    onClick: (route: NAVIGATION) => void
+    navigation: string
+}
+
+const Sidebar: Component<IProps> = (props) => {
+    return (
+        <div class="flex flex-col justify-between pr-[24px] pt-[10px]">
+            <div class="flex flex-col gap-[24px]">
+                <div
+                    onClick={() => {
+                        props.onClick(NAVIGATION.HOME)
+                    }}
+                    classList={{
+                        'hover:bg-[#354473] bg-[#2B375E] focus-visible:border-[#fff]':
+                            props.navigation === NAVIGATION.HOME,
+                    }}
+                    class="cursor-pointer rounded-[6px] p-[8px]">
+                    <div class="flex flex-col justify-center items-center gap-[10px]">
+                        <AiFillHome
+                            size={34}
+                            color={props.navigation === NAVIGATION.HOME ? '#9092FF' : '#192736'}
+                        />
+                        <p class="text-[14px] text-white font-normal leading-[20px] not-italic whitespace-nowrap text-center">
+                            Home
+                        </p>
+                    </div>
+                </div>
+                <div
+                    onClick={() => {
+                        props.onClick(NAVIGATION.DEFAULT)
```

Comment:
`DEFAULT` here doesn't really mean anything, maybe something like CONFIGURATION_WIZARD or ONBOARDING? Depends on what you want that view to do